File: program/find-smallest-element-in-a-matrix/FindSmallestElementInAMatrix.c
#include<stdio.h>
int main()
{
 int i,j,m,n;
 float a[10][10], sm;
 printf("Enter row and column size:\n");
 scanf("%d%d", &m, &n);
 printf("Enter matrix elements:\n");
 for(i=0;i< m;i++)
 {
  for(j=0;j< n;j++)
  {
   printf("a[%d][%d]=",i,j);
   scanf("%f", &a[i][j]);
  }
 }
 sm = a[0][0];
 for(i=0;i< m;i++)
 {
  for(j=0;j< n;j++)
  {
   if(a[i][j]< sm)
   {
    sm = a[i][j];
   }
  }
 }
 printf("Smallest element = %f\n", sm);

}

File: program/find-smallest-element-in-a-matrix/FindSmallestElementInAMatrix.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
class matrix
{
        int[,]x;
        matrix()
        {
            x = new int[,] { { 4, 7, 3 }, { 9, 4, 2 }, { 0, 6, 4 } };
        }
        void printarray()
        {
            Console.WriteLine("Elements in the given Matrix : ");
            for (int i = 0; i < 3; i++)
            {
                for (int j = 0; j < 3; j++)
                {
                    Console.Write(x[i, j] + "\t");
                }
                Console.WriteLine("\n");
            } 
        }
        int max()
        {
            int small = x[0, 0];
            for (int i = 0; i < 3; i++)
            {
                for (int j = 0; j < 3; j++)
                {
                    if (small > x[i, j])
                    {
                        small = x[i, j];
                    }
                }
            }
            return small;
        }
        public static void Main()
        {
            matrix obj = new matrix();
            obj.printarray();
            Console.WriteLine("Smallest Element in the Matrix : {0}", obj.max());
            Console.ReadLine();
        }
 }
